app/file_sender.py

```diff
@@ -18,40 +18,77 @@
 logger = wrap_logger(logging.getLogger(__name__))
 
 
-def process_complete_file(print_file: Path, pack_code: PackCode, batch_id, batch_quantity):
-    context_logger = logger.bind(pack_code=pack_code.value, batch_id=batch_id, batch_quantity=batch_quantity)
+def process_complete_file(complete_partial_file: Path, action_type: ActionType, pack_code: PackCode, batch_id,
+                          batch_quantity, context_logger):
     supplier = DATASET_TO_SUPPLIER[PACK_CODE_TO_DATASET[pack_code]]
 
     context_logger.info('Encrypting print file')
-    encrypted_print_file, filename = encrypt_print_file(print_file, pack_code, supplier)
+    encrypted_print_file, filename = encrypt_print_file(complete_partial_file, pack_code, supplier)
+
+    if is_file_over_size(encrypted_print_file):
+        context_logger.info('Encrypted file too large, splitting it into two',
+                            file_size_bytes=encrypted_print_file.stat().st_size)
+        split_partial_file(complete_partial_file, action_type, pack_code, batch_id, int(batch_quantity))
+        context_logger.info('File successfully split')
+        return
```

Comment:
This isn't removing the over sized encrypted file it has generated.

Reply:
Done